thewalrus/quantum/adjacency_matrices.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Functions for rescaling adjacency matrices as well as calculating the Qmat
+covariance matrix from an adjacency matrix.
+"""
+
+import numpy as np
+from scipy.optimize import root_scalar
+
+from .conversions import Xmat
+from .means_and_variances import mean_number_of_clicks_graph
+
+
+def find_scaling_adjacency_matrix_torontonian(A, c_mean):
```

Comment:
Minor: this function returns the scaling factor, rather than finding an adjacency matrix; maybe a better name is `adj_scaling_torontonian`?